ggml/src/ggml-hexagon/htp/binary-ops.c

```diff
@@ -113,10 +113,30 @@ static void binary_job_f32_per_thread(const struct htp_tensor * src0,
     uint8_t * restrict dst_ptr        = (uint8_t *) dst->data + (src0_start_row * dst_row_size);
 
     const uint8_t * restrict data_src1 = (const uint8_t *) src1->data;
-    const uint8_t * restrict src1_ptr  = NULL;
+
+    const uint32_t ne0201 = ne02 * ne01;
+    uint32_t       div03[2];
+    uint32_t       div02[2];
+    init_fastdiv_values(ne0201, &div03[0], &div03[1]);
+    init_fastdiv_values(ne01, &div02[0], &div02[1]);
+
+    uint32_t mod13[2];
+    uint32_t mod12[2];
+    uint32_t mod11[2];
+    init_fastdiv_values(ne13, &mod13[0], &mod13[1]);
+    init_fastdiv_values(ne12, &mod12[0], &mod12[1]);
+    init_fastdiv_values(ne11, &mod11[0], &mod11[1]);
```

Comment:
Liked @lhez said, maybe we could move this to cpu once tensor init, and save it at `htp_tensor`?

Reply:
Yep. The host would be even better.